pkgs/applications/editors/emacs/elisp-packages/manual-packages.nix

```diff
@@ -41,6 +39,8 @@ lib.packagesFromDirectoryRecursive {
   emacsSessionManagement = self.session-management-for-emacs;
   rectMark = self.rect-mark;
   sunriseCommander = self.sunrise-commander;
+
+  elpaca = callPackage ./manual-packages/elpaca { inherit (pkgs) git; };
```

Comment:
Put `elpaca` between `codeium` and `lsp-bridge`.  (This suggestion does not block the merge of this PR.)